### PR TITLE
Vermont capital gain exclusion

### PR DESCRIPTION
**Pull request by mzx-4real**
Relates #2624

### Summary
:sparkles::page_facing_up::wrench:

This pull request adds a new feature to the policy engine: the Vermont capital gain exclusion, which allows taxpayers to exclude a portion of their capital gains from their state income tax. It also renames and updates some existing variables and parameters related to Vermont adjusted gross income, and removes some unused files and references. It updates the changelog entry and adds tests for the new feature.

> _`Vermont` gains new rules_
> _Exclusion and subtraction_
> _Fall leaves bring changes_

### Walkthrough
*  Add Vermont capital gain exclusion feature to the policy engine ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
   * Create new parameter files for the cap, flat max, percentage max, and percentage ratio of the exclusion ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-1c77741df303b437dda0e12365f947c890bd588fc07b68348902666abe17e187R1-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-881f06a2f4aafeb8c5c00251c973450ab89c7fd7a7bb905154fb943203301d14R1-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-2033d8c79d59444acfeed813431af898a5cc0d42e1e1857148186b9c867c856cR1-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-f3444ca2069561ae843f3f4a961ad76b18e36b2198956a062adc04a5ea0f146eR1-R15))
   * Create new variable file for the exclusion amount, which depends on the adjusted net capital gain, taxable income, and state code ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-644b23df0dbc80903d3ec4d59c086f5e9b0f3a8a4fd0f8426eddb4dd69197d94R1-R38))
   * Create new test file for the exclusion amount, with five unit tests for different scenarios ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-5d379de6fb8ecabe8eed850f7f84cfe6f68f05cb6ee4475e54aa53c95ba2483fR1-R44))
* Update Vermont adjusted gross income variable and subtractions ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-c174b78e440841750174c9f041ab5510a7b82fe4158a8c204e567d494ce3bb78L14-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-e53b6fc7915e4489ef8722758b6e370a813ea98e387583dc2b78d97767d33a91R1-R18))
   * Rename `vt_agi_subtractions` variable to `vt_subtractions` to avoid confusion with `vt_agi_additions` ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-c174b78e440841750174c9f041ab5510a7b82fe4158a8c204e567d494ce3bb78L14-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-e53b6fc7915e4489ef8722758b6e370a813ea98e387583dc2b78d97767d33a91R1-R18), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-aa126afd3fd6d854e97157e4ca225c16c3a15dcee782e8b3b26957a4141419f8), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-3d3e0e0508b8f3ca30470933ed023041247e2764151f8a7753d06b4ff9d4a842))
   * Add new parameter file for the subtractions, which include the capital gain exclusion and other sources of income ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-8299b4ee7331e5c8c2dfaf432847d46016da57b0dc4fc86cb0cf63b94c52bd46R1-R19), [link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-e53b6fc7915e4489ef8722758b6e370a813ea98e387583dc2b78d97767d33a91R1-R18))
* Remove Vermont income tax variable from state income tax variable ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-af0d2b84ba0adacb1b6e826175576659344b80a34710502f7791da61cbd9c600L38))
   * Delete the reference to `vt_income_tax` from `state_income_tax`, as it is not yet implemented and would cause an error ([link](https://github.com/PolicyEngine/policyengine-us/pull/3006/files?diff=unified&w=0#diff-af0d2b84ba0adacb1b6e826175576659344b80a34710502f7791da61cbd9c600L38))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml:1]: rename to rate

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+description: Vermont uses this percentage to calculate capital gain exclusion cap.
+metadata:
+  label: Vermont capital gain exclusion cap calculation ratio
+  period: year
+  unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml:5]:
```suggestion
  unit: /1
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/flat/max.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/flat/max.yaml:1]: rename to max_amount

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/flat/max.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+description: Vermont caps the capital gain flat exclusion at this amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/flat/max.yaml:1]:
```suggestion
description: Vermont limits the flat capital gain exclusion at this amount.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+description: Vermont multiplies this percentage and adjusted net capital gain as the capital gain percentage exclusion.
+metadata:
+  label: Vermont capital gain percentage exclusion calculation ratio
+  period: year
+  unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml:5]:
```suggestion
  unit: /1
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/max.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/max.yaml:1]: rename to max _amount

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        flat_exclusion = min_(adjusted_net_capital_gain, p.flat.max)
+        # The percentage exclusion equals to 40% of the adjusted net capital gain and has a maximum value
+        percentage_exclusion = (
+            adjusted_net_capital_gain * p.percentage.percentage
+        )
+        percentage_exclusion = min_(percentage_exclusion, p.percentage.max)
+        # Filer can choose from flat or percentage exclusion. Assume the filer will always choose the larger one
+        chosen_exclusion = max_(flat_exclusion, percentage_exclusion)
+        # The chosen exclusion should not exceed 40% of federal taxable income
+        federal_taxable_income = tax_unit("taxable_income", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion.py:36]: it seems to be 40% of net capital gains: 
image

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml:1]: rename to rate

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+description: Vermont multiplies this percentage and adjusted net capital gain as the capital gain percentage exclusion.
+metadata:
+  label: Vermont capital gain percentage exclusion calculation ratio
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml:3]:
```suggestion
  label: Vermont capital gain percentage exclusion calculation rate
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml:1]: make an exclusions folder and rename the folder to capital_gain

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+description: Vermont uses this percentage to calculate capital gain exclusion cap.
+metadata:
+  label: Vermont capital gain exclusion cap calculation ratio
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/cap.yaml:3]:
```suggestion
  label: Vermont capital gain exclusion rate
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/capital_gain_exclusion/percentage/percentage.yaml:1]: one of the 0.4% seems to be applied to depreciable personal property - not sure if there is a variable for that
image

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion.py:1]: make a separate file for the non-flat exclusion

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  reference:
+    - title: 2022 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 18
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2022.pdf#page=1
+    - title: 2021 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 18
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2021.pdf#page=1
+     # Legal code specify maximum percentage exlcusion as $35,000. Date unknown.
+    - title: Legal code Titl. 32 V.S.A. § 5811(21)(B)(ii)(II)
+      href: https://legislature.vermont.gov/statutes/section/32/151/05811 
+values:
+  2021-01-01: 35_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml:15]:
```suggestion
  2021-01-01: 350_000
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+description: Vermont limits the flat capital gain exclusion at this amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:1]:
```suggestion
description: Vermont limits the flat capital gains exclusion at this amount.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:1]: everywhere, I missed that but this is the phrasing from the form

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/rate.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+description: Vermont multiplies this percentage and adjusted net capital gain as the capital gain percentage exclusion.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/rate.yaml:1]:
```suggestion
description: Vermont multiplies the adjusted net capital gains by this rate under the capital gain percentage exclusion.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml [hunk 10]
```diff
@@ -0,0 +1,15 @@
+description: Vermont uses this percentage to calculate capital gain exclusion cap.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml:1]:
```suggestion
description: Vermont applies this percentage to the flat or percentage capital gains exclusion.
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gain_exclusion.py [hunk 11]
```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class vt_capital_gain_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont capital gain exclusion"
+    unit = USD
+    documentation = "This is subtracted from federal adjusted gross income in Vermont as captial gain exclusion."
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gain_exclusion.py:9]:
```suggestion
    documentation = "Vermont as capital gain exclusion which is calculated either as a flat amount or as a percentage of adjusted net capital gains."
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml (hunk 8 above)
Comment (mzx-4real) [reply policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:1]: Should I change the variable names as well? or just the descriptions and labels.

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/subtractions.yaml [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  unit: list
+  label: Vermont subtractions
+  reference:
+    - title: Instruction for 2022 SCHEDULE IN-112 # PART 1 SUBTRACTIONS TO FEDERAL ADJUSTED GROSS INCOME
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf#page=1
+    - title: Instruction for 2021 SCHEDULE IN-112 # PART 1 SUBTRACTIONS TO FEDERAL ADJUSTED GROSS INCOME
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-112-2021.pdf#page=1
+    - title: Titl. 32 V.S.A. § 5811(21)(B)(i), (C)(iv), (B)(vi), (B)(ii), (B)(iv)
+      href: "https://legislature.vermont.gov/statutes/section/32/151/05811"
```
Comment (LeqiGogo) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/subtractions.yaml:15]:
```suggestion
      href: https://legislature.vermont.gov/statutes/section/32/151/05811
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_percentage_capital_gains_exclusion.py [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+    definition_period = YEAR
+    defined_for = StateCode.VT
+    reference = (
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-153-2022.pdf#page=1"  # 2022 Schedule IN-153 Vermont Capital Gains Exclusion Calculation
+        "https://legislature.vermont.gov/statutes/section/32/151/05811"  # Titl. 32 V.S.A. § 5811(21)(B)(ii)
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-153%20Instr-2022.pdf"
+    )
+
+    def formula(tax_unit, period, parameters):
+        # Get adjusted net capital gain
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_percentage_capital_gains_exclusion.py:19]:
```suggestion
        # Get adjusted net capital gain, which is capped at 0
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+    definition_period = YEAR
+    defined_for = StateCode.VT
+    reference = (
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-153-2022.pdf#page=1"  # 2022 Schedule IN-153 Vermont Capital Gains Exclusion Calculation
+        "https://legislature.vermont.gov/statutes/section/32/151/05811"  # Titl. 32 V.S.A. § 5811(21)(B)(ii)
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-153%20Instr-2022.pdf"
+    )
+
+    def formula(tax_unit, period, parameters):
+        # Get adjusted net capital gains
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py:19]:
```suggestion
        # Get adjusted net capital gains, which is capped at 0
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+        adjusted_net_capital_gain = tax_unit(
+            "adjusted_net_capital_gain", period
+        )
+        p = parameters(
+            period
+        ).gov.states.vt.tax.income.agi.exclusions.capital_gain
+        # The flat exclusion is the less of $5,000 or the actual amount of net adjusted capital gains
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py:26]:
```suggestion
        # The flat exclusion is the less of a capped amount or the actual amount of net adjusted capital gains
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_percentage_capital_gains_exclusion.py [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+        adjusted_net_capital_gain = tax_unit(
+            "adjusted_net_capital_gain", period
+        )
+        p = parameters(
+            period
+        ).gov.states.vt.tax.income.agi.exclusions.capital_gain
+        # The percentage exclusion equals to 40% of the adjusted net capital gain and has a maximum value
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_percentage_capital_gains_exclusion.py:26]:
```suggestion
        # The percentage exclusion equals to a percentage of the adjusted net capital gain and has a maximum value
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml [hunk 17]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Vermont capital gains percentage exclusion max amount
+  period: year
+  unit: currency-USD
+  reference:
+    - title: 2022 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 18
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2022.pdf#page=1
+    - title: 2021 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 18
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2021.pdf#page=1
+     # Legal code specify maximum percentage exlcusion as $350,000. Date unknown.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml:11]:
```suggestion
     # Legal code specify the maximum percentage exclusion amount Date unknown.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+    - title: Legal code Titl. 32 V.S.A. § 5811(21)(B)(ii)(II)
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/percentage/max_amount.yaml:12]:
```suggestion
    - title: Legal code Titl. 32 V.S.A. § 5811(21)(B)(ii)
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Vermont flat capital gains exclusion max amount
+  period: year
+  unit: currency-USD
+  reference:
+    - title: 2022 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 9
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2022.pdf#page=1
+    - title: 2021 Schedule IN-153Vermont Capital Gains Exclusion Calculation # Line 9
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-153-2021.pdf#page=1
+     # Legal code specify maximum flat exlcusion as $5,000. Date unknown.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:11]:
```suggestion
     # Legal code specify maximum flat exclusion as $5,000. Date unknown.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/subtractions.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/subtractions.yaml:1]: leave the existing subtractions and add the vt_capital_gains_exclusion to it

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:1]: rename cap, same with percentage/max_amount

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml [hunk 20]
```diff
@@ -0,0 +1,15 @@
+description: Vermont limits the flat capital gains exclusion at this amount.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:1]:
```suggestion
description: Vermont caps the flat capital gains exclusion at this amount.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml [hunk 21]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Vermont flat capital gains exclusion max amount
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml:3]:
```suggestion
  label: Vermont flat capital gains exclusion cap
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml:1]: rename income_share_cap.yaml

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml [hunk 22]
```diff
@@ -0,0 +1,16 @@
+description: Vermont caps the net capital gains exclusion at the rate of the federal taxable income.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml:1]:
```suggestion
description: Vermont caps the net capital gains exclusion at this fraction of federal taxable income.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml [hunk 23]
```diff
@@ -0,0 +1,16 @@
...
+
+metadata:
+  label: Vermont capital gains exclusion rate
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/rate.yaml:4]:
```suggestion
  label: Vermont capital gains exclusion cap as fraction of taxable income 
```

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+
+    def formula(tax_unit, period, parameters):
+        # Get adjusted net capital gains, which is capped at 0
+        adjusted_net_capital_gain = tax_unit(
+            "adjusted_net_capital_gain", period
+        )
+        p = parameters(
+            period
+        ).gov.states.vt.tax.income.agi.exclusions.capital_gain
+        # The flat exclusion is the less of a capped amount or the actual amount of net adjusted capital gains
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py:26]: limit comment lines to 79 characters

File: policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py [hunk 25]
```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class vt_capital_gains_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont capital gains exclusion"
+    unit = USD
+    documentation = "Vermont as capital gains exclusion which is calculated either as a flat amount or as a percentage of adjusted net capital gains."
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_capital_gain_exclusion/vt_capital_gains_exclusion.py:9]:
```suggestion
    documentation = "Vermont excludes a portion of capital gains, calculated either as a flat amount or as a fraction of adjusted net capital gains, and limited by a fraction of federal taxable income."
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**